src/Controls/src/Core/Compatibility/Handlers/NavigationPage/iOS/NavigationRenderer.cs

```diff
@@ -1105,6 +1105,11 @@ public override void DidShowViewController(UINavigationController navigationCont
 			{
 				if (_navigation.TryGetTarget(out NavigationRenderer r))
 				{
+					if (r.NavigationBar is MauiNavigationBar navBar && navBar.TitleBarNeedsRefresh)
```

Comment:
Can this whole block get moved into `MauiNavigationBar`?

Then we aren't duplicating the code